Android/app/src/main/res/xml/intro_motion_scene.xml

```diff
@@ -0,0 +1,250 @@
+<?xml version="1.0" encoding="utf-8"?>
+<MotionScene xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:motion="http://schemas.android.com/apk/res-auto">
+
+    <Transition
+        motion:constraintSetEnd="@id/animation_end"
+        motion:constraintSetStart="@id/animation_start"
+        motion:duration="2000"
+        motion:motionInterpolator="easeInOut" />
+
+    <ConstraintSet android:id="@+id/animation_start">
+
+        <Constraint android:id="@id/hand_watermark"
+            motion:visibilityMode="ignore">
+            <Layout
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                motion:layout_constraintStart_toStartOf="parent"
+                motion:layout_constraintEnd_toEndOf="parent"
+                motion:layout_constraintBottom_toBottomOf="parent" />
+            <CustomAttribute
+                motion:attributeName="crossfade"
+                motion:customFloatValue="0.0" />
```

Comment:
If we remove the background and the `altSrc`, we could change this to:
```
            <CustomAttribute
                motion:attributeName="ColorFilter"
                motion:customColorValue="#272727" />
```

Reply:
Awesome! This is much better!